third_party/nvfuser/csrc/compute_at_map.cpp

```diff
@@ -503,6 +521,17 @@ void IterDomainGraph::build(Fusion* fusion) {
             }
           }
         }
+
+        // Mostly the same as the above for the permissive map but
+        // nothing to do for the loop map
+        for (auto& dset : permissive_resize_disjoint_sets.disjointSets()) {
+          auto& vec = dset->vector();
+          for (auto i : c10::irange(vec.size())) {
```

Comment:
They should be based on the most permissive mappings, so yes, they should be created using the permissive-resize map. See 27c867f11f81b096066d5f526be0526b2ea9640d.

This would also be an issue if we would end up having multiple maps that there may not be ordering of permissiveness.